File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/entities/adventurers/units/Juggernaut.java
package it.paranoidsquirrels.idleguildmaster.storage.data.entities.adventurers.units;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.entities.Skills;
import it.paranoidsquirrels.idleguildmaster.storage.data.entities.adventurers.Adventurer;
import it.paranoidsquirrels.idleguildmaster.storage.data.entities.adventurers.PotionDrinkerType;

/* JADX INFO: loaded from: classes3.dex */
public class Juggernaut extends Adventurer {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.entities.adventurers.Adventurer
    protected void configureStatistics() {
        this.maxLevel = 30;
        this.baseMaxHp = 215;
        this.baseConstitution = 28;
        this.baseIntelligence = 14;
        this.baseDexterity = 9;
        this.baseDefense = 25;
        this.baseMagicDefense = 25;
        this.threat = 4;
        this.imageId = R.drawable.unit_juggernaut;
        this.idName = R.string.adventurer_juggernaut_name;
        this.idDescription = R.string.adventurer_juggernaut_description;
        this.passiveSkill = Skills.PASSIVE_THREATENING_III;
        this.activeSkill = Skills.ACTIVE_TAUNT_II;
        this.weaponType = R.string.type_sword;
        this.armorType = R.string.type_armor_heavy;
        this.potionDrinkerType = PotionDrinkerType.WARRIOR;
        this.nextClasses.add("Titan");
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/entities/adventurers/units/Juggernaut.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.entities.adventurers.units

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.entities.Skills
import it.paranoidsquirrels.idleguildmaster.storage.data.entities.adventurers.Adventurer
import it.paranoidsquirrels.idleguildmaster.storage.data.entities.adventurers.PotionDrinkerType

class Juggernaut : Adventurer() {
    override fun configureStatistics() {
        maxLevel = 30
        baseMaxHp = 215
        baseConstitution = 28
        baseIntelligence = 14
        baseDexterity = 9
        baseDefense = 25
        baseMagicDefense = 25
        threat = 4
        imageId = R.drawable.unit_juggernaut
        idName = R.string.adventurer_juggernaut_name
        idDescription = R.string.adventurer_juggernaut_description
        passiveSkill = Skills.PASSIVE_THREATENING_III
        activeSkill = Skills.ACTIVE_TAUNT_II
        weaponType = R.string.type_sword
        armorType = R.string.type_armor_heavy
        potionDrinkerType = PotionDrinkerType.WARRIOR
        nextClasses.add("Titan")
    }
}
